open-api/rest-catalog-open-api.yaml

```diff
@@ -3142,6 +3162,10 @@ components:
           type: object
           additionalProperties:
             type: string
+        storage-credentials:
```

Comment:
Just curious why should return multiple credentials for one table? Any use cases?  The current TableOps just support one fileIO and also noticed the AWS and GCS credential refresh handler just pick the first credential from the server.

Reply:
The use case for this is outlined in https://github.com/apache/iceberg/pull/10722#discussion_r1755423596.